### ISSUE TITLE
How to set learning rate decay strategy and the related parameters.

### ISSUE THREAD

**Issue by lcy-seso** [PaddlePaddle/Paddle#3839]
In V1 version, a user can use the following method to set learning rate decay (decrease the learning rate as training goes on ):

```python
Settings(algorithm='sgd',
         batch_size=128,
         learning_rate=0.1/128,
         learning_method='momentum',
         learning_rate_decay_a=0.1,
         learning_rate_decay_b=400000 * 50,
         learning_rate_schedule="discexp", )
``` 
I am not sure how to set `learning_rate_schedule` , ` learning_rate_decay_b` and ` learning_rate_decay_b` , in v2 API?

**Comment by typhoonzero** [comment]
Try pass `kwargs` to optimzer's ctor?

```python
import paddle.trainer_config_helpers.optimizers as v1_optimizers
class Optimizer(object):
    def __init__(self, **kwargs):
        import py_paddle.swig_paddle as swig_api
        if 'batch_size' in kwargs:
            del kwargs['batch_size']  # not important for python library.

        def __impl__():
            v1_optimizers.settings(batch_size=1, **kwargs)

        self.__opt_conf_proto__ = config_parser_utils.parse_optimizer_config(
            __impl__)
        self.__opt_conf__ = swig_api.OptimizationConfig.createFromProto(
            self.__opt_conf_proto__)

```

**Comment by lcy-seso** [comment]
It can be done like this:
```python
optimizer = paddle.optimizer.Adam(
        learning_rate=1e-3,
        regularization=paddle.optimizer.L2Regularization(5e-4),
        learning_rate_decay_a=0.1,
        learning_rate_decay_b=400000 * 10,
        learning_rate_schedule="discexp")
```

**Comment by lcy-seso** [comment]
@typhoonzero You are right,  Thank you ~
